grammars/level12-Additions.lark

```diff
@@ -24,8 +7,6 @@ error_nested_define: define
 define: _DEFINE function_name (_SPACE _WITH arguments)? _EOL (_SPACE command) (_EOL _SPACE command)* _EOL? _END_BLOCK
```

Comment:
Calls are only allowed on this level as command right? Not as part of expression